resources/quality/weedo_x40/0.4_generic_ABS_Extra_Fine.inst.cfg

```diff
@@ -0,0 +1,38 @@
+[general]
+version = 4
+name = Default - Extra Fine
```

Comment:
Why prepend all the names with default? If everything is default, it doesn't change anything to add it.

I'd suggest renaming this to just "Extra Fine"